Vector.cpp

```diff
@@ -290,7 +392,7 @@ TEST(Vector_Test, constructorByInitList)
     }
 
     {
-        std::initializer_list<int> tmp = {1,2,3,4,5,6,7,8,9,10};
+        std::initializer_list<int> tmp = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```

Comment:
VSCode has C++ extension pack. You can use a build-in C++ formatter to avoid these formatting issues. There is the "Format on save" option. I found it very useful.